src/Traits/Translatable.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace TCG\Voyager\Traits;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Relations\Relation;
+use TCG\Voyager\Models\Translation;
+use TCG\Voyager\Translator;
+
+trait Translatable
+{
+    /**
+     * Check if this model can translate.
+     *
+     * @return bool
+     */
+    public function translatable()
+    {
+        if (isset($this->translatable) && $this->translatable == false) {
+            return false;
+        }
+
+        return !empty($this->getTranslatableAttributes());
+    }
+
+    /**
+     * Load translations relation.
+     *
+     * @return mixed
+     */
+    public function translations()
+    {
+        return $this->hasMany(Translation::class, 'foreign_key', $this->getKeyName())
+            ->where('table_name', $this->getTable())
+            ->whereIn('locale', config('voyager.multilingual.locales', []));
```

Comment:
The default language is `en` and is not stored in `translations` but in the model itself.

Let's say a model `Post` have been there for quite a long time, and all it's content is in `en`, then the default language should be set to `en` in the configuration and then they can be translated into other languages.
The other languages will however be stored inside the `translations` table.
So if the user was ever to disable multilingual on the site, they would still have the original value of their posts in their main language.